src/app/help/api/page.tsx

```diff
@@ -0,0 +1,500 @@
+"use client";
+
+import ReactMarkdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+import { CopyMarkdownButton } from "@/components/CopyMarkdownButton";
+import { EXTERNAL_URLS } from "@/lib/constants";
+
+const apiDocumentation = `# API Documentation
+
+Roast My Post provides a RESTful API for programmatic access to document evaluation features.
```

Comment:
This needs a large warning on top saying that this is all experimental and might quickly change. We don't recommend relying on this much at this point.